### PR TITLE
explicitly register commands as services, so that support Symfony 4

### PR DESCRIPTION
**Pull request by tiger-seo**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/.idea/
```
Comment (saimaz) [inline .gitignore]: can you remove .idea in this file?

**Comment by tiger-seo** [comment]
@saimaz done

**Comment by saimaz** [comment]
Thanks for the PR @tiger-seo

**Comment by trandangtri** [comment]
Hi @saimaz,
With this PR, we won't support Symfony 2.8 and older, is it right?

CC @tiger-seo

**Comment by saimaz** [comment]
yes, by releasing a 5.1 we can drop support for the Symfony 2.8

**Pull request merged**